### DISCUSSION TITLE
High-quality sound Engine

### DISCUSSION THREAD

**Discussion by LGHRM** [stakira/OpenUtau#1054] [Ideas]
I would like the feature that if the recorded audio of UTAU VB is of high quality, such as 24bit/48kHz, the output audio will be of high quality as well. (Sorry for DeepL English)

**Comment by nagotown** [comment]
this is dependent on resamplers, meaning every resampler dev would have to add support, which is not likely as most resamplers have ceased development

**Comment by BagelHero** [comment]
What nagotown said is true... However, it is possible to implement new renderers like WORLDLINE-R... Perhaps a new renderer might be able to have support for variable bitrates. Unfortunately, I think the difference in audio output quality would be minimal as most of the quality issues associated with UTAU are to do with the method of re-timing and re-pitching samples, rather than the samples themselves.
